pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -196,6 +196,9 @@ type ShootCredentialsRotation struct {
 	// SSHKeypair contains information about the ssh-keypair credential rotation.
 	// +optional
 	SSHKeypair *ShootSSHKeypairRotation `json:"sshKeypair,omitempty" protobuf:"bytes,3,opt,name=sshKeypair"`
+	// ObservabilityUser contains information about the observability-user credential rotation.
+	// +optional
+	ObservabilityUser *ShootObservabilityUserRotation `json:"observabilityUser,omitempty" protobuf:"bytes,4,opt,name=observabilityUser"`
```

Comment:
```suggestion
	Observability *ShootObservabilityRotation `json:"observability,omitempty" protobuf:"bytes,4,opt,name=observability"`
```

Remove `User` from name and adapt the `json` tags. Same for `v1beta1`